.gitattributes

```diff
@@ -27,3 +27,4 @@
 *.xls binary
 *.xlsx binary
 *.odp binary
+*.zip binary
```

Comment:
Let's add this line in .gitattributes in optaplanner and optalanner-quickstarts too